orchagent/neighorch.cpp

```diff
@@ -271,6 +272,18 @@ bool NeighOrch::getNeighborEntry(const IpAddress &ipAddress, NeighborEntry &neig
     return false;
 }
 
+bool NeighOrch::getNeighborEntry(const IpAddress &ipAddress, NeighborEntry &neighborEntry, MacAddress &macAddress)
```

Comment:
Yes, now only mirror session use it. There is dest ip given, we have to find out ifname. To support vrf for mirror, it would be ip and vrf in configuration, then add a vrf argument.

Reply:
If `getNeighborEntry `is invoked with only IP address and not `Ifname`, then `getRouterIntfsAlias `must only check for interfaces in default VRF.

Reply:
That's right.